dataland-frontend/src/components/layout/TheFooter.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <footer class="footer">
+    <div class="footer__left">
+      <div class="footer__logo" role="img" aria-label="Footer Logo">
+        <img
+          v-for="(img, index) in footerSection?.image ?? []"
+          :key="index"
+          :src="img"
+          :alt="footerSection?.text.join(' ')"
+          class="footer__logo-img"
+        />
+      </div>
+      <div class="footer__copyright">{{ footerText }}</div>
+    </div>
+
+    <nav class="footer__right" role="navigation" aria-label="Footer Navigation">
+      <router-link to="/legal" class="footer__right-link" aria-label="Legal">Legal</router-link>
```

Comment:
This one leads to an error page.

Reply:
The content of the page is still in progress. I remove the link for now.

Reply:
Ok thx.